articles/extensions/delegated-admin.md

```diff
@@ -130,44 +357,23 @@ There are two available views, *Users* and *Logs*. At the *Users* view you can s
     </tr>
 </table>
 
-Notice the new *Reset password* option available via the extension. This option will send an email to the user allowing them to choose a new password. To do this click on a user and select *Actions > Reset password*.
+Notice the new *Reset Password* option available via the extension. This option will send an email to the user allowing them to choose a new password. To do this click on a user and select *Actions > Reset Password*.
 
 ![](/media/articles/extensions/delegated-admin/reset-pass-01.png)
 
 This will send an email to the user, containing a link to change the password.
 
-At the *Logs* view you can see log data of authentications made by your users. The contents of this view are a subset of the data displayed in the [Logs dashboard](${manage_url}/#/logs), which also displays data for the actions taken in the dashboard by the administrators.
-
+At the *Logs* view you can see log data of authentications made by your users (this tab is only visible to users with the `Delegated Admin - Administrator` role). The contents of this view are a subset of the data displayed in the [Logs Dashboard](${manage_url}/#/logs), which also displays data for the actions taken in the dashboard by the administrators.
 
 ## Customize the dashboard
 
-You can use the **Title** and **Custom_CSS** variables to customize the look and feel of your dashboard. 
+You can use the **Title** and **Custom_CSS** variables to customize the look and feel of your dashboard.
 
 Navigate to the [Extensions](${manage_url}/#/extensions) page and go to the settings of the **Delegated Administration** extension. We are going to set a custom title and a custom css file:
-- Set the **Title** to `Fabrikam User Management`.
-- Set the **Custom_CSS** to `https://rawgit.com/auth0-extensions/auth0-delegated-administration-extension/master/docs/theme/fabrikam.css`.
+
+- Set the **TITLE** to `Fabrikam User Management`.
+- Set the **CUSTOM_CSS** to `https://rawgit.com/auth0-extensions/auth0-delegated-administration-extension/master/docs/theme/fabrikam.css`.
 
 Save your changes, navigate to the extension and login. The dashboard now looks like that:
 
 ![](/media/articles/extensions/delegated-admin/custom-dashboard.png)
```

Comment:
Done (with kelly user). Not sure if this is correct, it looked custom, while sandrino user has the auth0 ui.